specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2021-04-01-preview/examples/CosmosDBDataTransferServiceCreate.json

```diff
@@ -0,0 +1,39 @@
+{
+  "parameters": {
+    "accountName": "ddb1",
+    "resourceGroupName": "rg1",
+    "api-version": "2021-04-01-preview",
+    "subscriptionId": "subid",
+    "serviceName": "DataTransfer",
+    "servicesCreateUpdateParameter": {
+      "properties": {
+        "instanceSize": "Cosmos.D4s",
+        "instanceCount": 1
+      }
+    }
+  },
+  "responses": {
+    "202": {},
+    "200": {
+      "body": {
+        "id": "/subscriptions/subid/resourceGroups/rg1/providers/Microsoft.DocumentDB/databaseAccounts/ddb1/services/DataTransfer",
+        "name": "DataTransfer",
```

Comment:
>"name": "DataTransfer", [](http://example.com/codeflow?start=8&length=23)

How does the server knows if this is to create a data trasnfer service or the other one? Is it by the resource name? Should the request body contain serviceType?

Reply:
yes the request body should contain serviceType. ServiceType tells whether it is DataTransfer or SqlDedicatedGateway